tools/gap2seq.py

```diff
@@ -0,0 +1,110 @@
+'''
+    Gap2Seq - assembly gap closing tool
+'''
+
+import logging
+import os
+import os.path
+import subprocess
+import shlex
+import shutil
+import tempfile
+import time
+
+import Bio.Seq
+
+import tools
+import tools.samtools
+import util.file
+import util.misc
+
+TOOL_NAME = 'gap2seq'
+TOOL_VERSION = '2.1'
+
+log = logging.getLogger(__name__)
+
+class Gap2SeqTool(tools.Tool):
+    """Tool wrapper for the Gap2Seq gap-closing tool."""
+
+    def __init__(self, install_methods=None):
+        if install_methods is None:
+            install_methods = [tools.CondaPackage(TOOL_NAME, version=TOOL_VERSION, executable='Gap2Seq.sh')]
+        tools.Tool.__init__(self, install_methods=install_methods)
+
+    def version(self):
+        return TOOL_VERSION
+
+    def execute(self, args):    # pylint: disable=W0221
+        tool_cmd = [self.install_and_get_path()] + args
+        log.debug('running gap2seq: ' + ' '.join(tool_cmd))
+        subprocess.check_call(tool_cmd)
```

Comment:
Right, I was thinking of masking crashes since it's a non-critical path in the pipeline; on the other hand, if crashes are very rare then getting alerted to them might not be bad.  I haven't actually seen it crash.